packages/camera/src/components/Controls/__tests__/AddDamageButton.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
+import '@testing-library/jest-dom';
+import AddDamageButton from '../AddDamageButton';
+
+describe('Add Damage Button component', () => {
+  it('renders the correct message', () => {
+    render(<AddDamageButton label="add damage button" customStyle={{}} />);
+    expect(screen.getByText('add damage button')).toBeTruthy();
```

Comment:
From what I understand, `screen.getBy...` will **never** not be truthy. The method will either :

* Return the element
* Throw if no element (or more than one) is found

So instead of `expect(...).toBeTruthy();` you should assert that the call will not throw. You could do something like :

```javascript
expect(() => screen.getByText('add damage button')).not.toThrow();
```

Reply:
Yeah! It can be another matcher like "to be in the document" something like so....

Reply:
used toBeOnTheScreen()